nnpdf_data/nnpdf_data/commondata/CMS_2JET_7TEV/metadata.yaml

```
@@ -53,10 +53,18 @@ implemented_observables:
   data_central: data.yaml
   data_uncertainties:
   - uncertainties.yaml
-
+  variants:
+    legacy:
```

```suggestion
    legacy_theory:
```

I think this makes a bit clear that we only have a legacy difference for the theory.